code/src/components/AddBook.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import styled from "styled-components";
+
+import { toread } from "../reducers/toread";
+
+const AddContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #3C4A6B;
+  padding: 100px 0;
+`;
+
+const Input = styled.input`
+  border: none;
+  border-bottom: 6px solid #2A344A;
+  background: #3C4A6B;
+  font-size: 20px;
+  width: 75%;
+  margin-bottom: 20px;
+  padding: 10px 0;
+  text-align: center;
+  font-family: "Josefin Sans", sans-serif, "Courier New", monospace;
+  color: #FDFFF3;
+
+  &:focus {
+    outline: 2px solid #2A344A;
+  }
+
+  @media (min-width: 768px) {
```

Comment:
Good work with the responsiveness in styled components.
It can be good to have an order/structure in the css code as well. Ex. flex and dimensions (margins and paddings) are placed before font-size, color and other more detailed code. It will be easier to read.